src/test/FuzzerImpl.cpp

```diff
@@ -909,23 +909,31 @@ applyFuzzOperations(LedgerTxn& ltx, PublicKey const& sourceAccount,
 // Unlike Asset, this can be a constexpr.
 struct AssetID
 {
-    constexpr AssetID() : mIsNative(true), mId(0)
+    constexpr AssetID() : mIsNative(true), mIssuer(0), mSuffixDigit(0)
     {
     }
 
-    constexpr AssetID(int id) : mIsNative(false), mId(id)
+    constexpr AssetID(int id) : AssetID(id, id)
```

Comment:
No, it's just convenient with the current configuration where many assets are of that form.  It could become obsolete with a different configuration, but I didn't want to churn that more than necessary in this PR.